examples/spark/spark-gluster/spark-master-service.yaml

```diff
@@ -0,0 +1,13 @@
+kind: Service
+apiVersion: v1
+metadata: 
+  name: spark-master
+  labels: 
+    name: spark-master
```

Comment:
Do not use "name" as a label key. That has been a source of user confusion in the past.